specification/maps/resource-manager/Microsoft.Maps/preview/2024-07-01-preview/maps-management.json

```diff
@@ -0,0 +1,1201 @@
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CreatorNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Maps Creator will be deleted."
+          },
+          "204": {
+            "description": "The specified Maps Creator resource was not found. Nothing was deleted."
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Creators"
+        ],
+        "description": "Get a Maps Creator resource.",
+        "operationId": "Creators_Get",
+        "x-ms-examples": {
+          "Get Creator Resource": {
+            "$ref": "./examples/GetMapsCreator.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CreatorNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful.",
+            "schema": {
+              "$ref": "#/definitions/Creator"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "AccountNameParameter": {
+      "name": "accountName",
+      "in": "path",
+      "description": "The name of the Maps Account.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "minLength": 3,
+      "maxLength": 98,
+      "pattern": "^[a-zA-Z][a-zA-Z0-9-]{3,98}[a-zA-Z0-9]$"
```

Comment:
>  "pattern": "^[a-zA-Z][a-zA-Z0-9-]{3,98}[a-zA-Z0-9]$" [](http://example.com/codeflow?start=4&length=54)

What happens to the accountNames that were created using earlier API versions. 
Can they not use the new API version now ? How are those accountNames handled eventually ?

Reply:
Updated the pattern to use same as previous versions. No change applied to accountNames handling.